dvc/scm/git/__init__.py

```diff
@@ -162,6 +163,13 @@ def _add_entry_to_gitignore(entry, gitignore):
                 prefix = "" if last == "\n" else "\n"
             fobj.write("{}{}\n".format(prefix, entry))
 
+    @staticmethod
+    def _escape(entry):
```

Comment:
I would prefer a function over static method.

Reply:
@Suor , on the same module? I agree :+1:

Reply:
@Suor , there's a PR that will move this method to the pathspec module  https://github.com/cpburnz/python-path-specification/pull/27

Reply:
We don't know when that or would be merged. There is no reason making it a static method anyway.

Reply:
@Suor Even deepsource is not complaining about that one, it is just a matter of preference here :slightly_smiling_face: About the PR to pathspec, the author showed clear interest, so I'm pretty sure the patch will be merged in one form or another.

Reply:
@Suor Aand that PR got merged. 🙂